examples/camera_params.py

```diff
@@ -24,7 +24,7 @@ async def run():
     drone = System()
     await drone.connect(system_address="udpin://0.0.0.0:14540")
 
-    tasks = [
+    _tasks = [
```

Comment:
It is unusual to assemble these tasks but not await them.  Are we certain that they will be run at some point?

Reply:
Well, so this code was working fine but ruff complained that the tasks were dangling: https://docs.astral.sh/ruff/rules/asyncio-dangling-task/.

So, the advice was to create a strong reference to the tasks. However, now ruff complains that tasks are assigned but never used. It feels like you can't win.

Reply:
I will re-review this evening.

Reply:
I was wrong...  This usage is correct.
```
>>> import asyncio
>>> async def print_char(char: str) -> None:
...     print(char)
...
>>> async def run(chars: str) -> None:
...     tasks = [asyncio.create_task(print_char(char)) for char in chars]
...
>>> asyncio.run(run("abcdefg"))
a
b
c
d
e
f
g
```